src/SmartNumber.php

```diff
@@ -43,27 +52,39 @@ final class SmartNumber
 	/** @var string */
 	private $input;
 
-	/** @var string */
-	private $string;
-
-	/** @var BigInteger */
-	private $integer;
-
 	/** @var BigDecimal */
 	private $decimal;
 
 	/** @var FractionNumbersOnly|null */
 	private $fraction;
 
+	/**
+	 * Possible thousand separators (first is default)
+	 *
+	 * @var string[]|null
+	 */
+	private $thousandsSeparators;
+
+	/**
+	 * Possible decimal point signs (first is default)
+	 *
+	 * @var string[]|null
+	 */
+	private $decimalPointSigns;
+
 
 	/**
 	 * @param int|null $accuracy
 	 * @param string $number number or real user input
+	 * @param string[] $decimalPointSigns
+	 * @param string[] $thousandsSeparators
 	 * @throws NumberException
 	 */
-	public function __construct(?int $accuracy, string $number)
+	public function __construct(?int $accuracy, string $number, array $decimalPointSigns = ['.', ','], array $thousandsSeparators = ['', ' '])
 	{
 		$this->accuracy = $accuracy ?? 100;
+		$this->decimalPointSigns = $decimalPointSigns;
+		$this->thousandsSeparators = $thousandsSeparators;
```

Comment:
I think this configuration should be defined in class public constant.

Why you want to configure it dynamically?

Reply:
Removed.

Reply:
Thanks, more better.